syslog-ng/main.c

```diff
@@ -208,6 +208,8 @@ main(int argc, char *argv[])
   GOptionContext *ctx;
   GError *error = NULL;
 
+  timeutils_setup_timezone_hook();
+
   MainLoop *main_loop = main_loop_get_instance();
```

Comment:
please move this right after app_startup(). should be just as good, unless I am missing something. This probably works here, but "feels" a bit too early. I don't think it is important to cover our command line parsing.